### PR TITLE
doc: add some doc for server

### PR DESCRIPTION
**Pull request by anda-ren**
## Description
add some doc for server
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [x] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: docs/docs/guides/config/server_admin.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Server Instance superuser Password Reset
```
Comment (tianweidut) [inline docs/docs/guides/config/server_admin.md:2]: update title? is `Controller Admin Settings` good for the title?

Comment (anda-ren) [reply docs/docs/guides/config/server_admin.md]: done

File: docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/evaluation.md [hunk 2]
```diff
@@ -53,3 +53,29 @@ title: Starwhale Evaluation-模型评测
 - task：task是最终运行的实体。在Cloud Instance中，一个task就是一个Pod的container；在Standalone Instance中，一个task就是一个Python Thread。
 
 job-step-task 的抽象是实现 `Starwhale Evaluation` 分布式运行的基础。
+
+## 4. 在服务器上运行评测任务
+
+### 覆盖step配置
+
+你可以在创建job的时候提交一个yaml来覆盖在`SWPM`中硬编码的step配置。`resources`字段的行为与[K8S容器与pod的资源管理](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)中描述的一样。
```
Comment (tianweidut) [inline docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/evaluation.md:61]: `SWPM` -> `swmp` ?

Comment (anda-ren) [reply docs/i18n/zh/docusaurus-plugin-content-docs/current/guides/evaluation.md]: done

**Review by tianweidut — APPROVED** [review]

**Pull request merged**